src/com/sfuentes/OnlyPositive.java

```diff
@@ -0,0 +1,18 @@
+package com.sfuentes;
+
+import java.util.ArrayList;
+
+public class OnlyPositive {
+  public ArrayList<Double> showOnlyPositiveNumbers(double[] numbers) {
+
+    ArrayList<Double> positive = new ArrayList<>();
```

Comment:
(not required) rename the variable to positives in plural since it is a list.

Reply:
Fixed it